server/rest.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
+
+const { init } = require('./queries');
+
+let Queries = null;
+
+const start = app => {
+  Queries = init();
+
+  /* root */
+
+  app.get('/', async (res) => {
+    res.redirect('https://kpi-team-five.github.io/TeamFive/app/index.html');
+  });
+  
+  /* auth */
+  app.post('/auth', async (req, res) => {
+    const body = req.body;
+    const user = { id: null, login: body.login, pass: body.pass }
+    const users = await Queries.UserQueries.getAll();
+    let result = false;
+    for (const entry of users) {
+      if (entry.login == user.login && entry.pass == user.pass) {
+        result = true;
+        break;
+      }
+    }
+    return res.send(result);
+  });
+
+  /* users */
+  app.post('/users', async (req, res) => {
+    const body = req.body;

```

Comment:
```suggestion
    const { body } = req;
```
Use destructuring assignment